lib/main.mlx

```diff
@@ -54,15 +54,15 @@ let getting_started ~install_url () =
       "Run"
     </h4>
     <Code>
-"$ PROJECT=hello_world"
-"$ dune init proj \"$PROJECT\""
-"$ cd \"$PROJECT\""
-"$ dune pkg lock"
-"$ dune exec \"$PROJECT\""
+{|$ PROJECT=hello_world|}
+{|$ dune init proj "$PROJECT"|}
+{|$ cd "$PROJECT"|}
+{|$ echo "(dune-lang 3.21)\n(pkg enabled)" > dune-workspace|}
```

Comment:
So it less mysterious what we asking folks to do here, and to make it easy for people to opt out if they want.

```suggestion
{|# To opt in to using dune for package management |}
{|$ echo "(dune-lang 3.21)\n(pkg enabled)" > dune-workspace|}
```